### DISCUSSION TITLE
Best way to undo a bad import?

### DISCUSSION THREAD

**Discussion by maelstrm-code** [simulot/immich-go#513] [Q&A] [answered]
I accidentally put the wrong path in to immich-go and imported a stack of rubbish like screenshots, icons etc. What is the best way to delete the thousand or so images that were uploaded in a burst on a specific date range? I have the file names in the log, but Immich doesn't seem to natively allow filtering by upload date in the UI. I'd really like to avoid nuking Immich and starting again, but that seems to be my best option.

**Comment by simulot** [comment, marked as answer]
Unfortunately, there isn't a practical way to reverse the import.

**Comment by redge76** [comment]
Tags are comming and I hope it will be possible to automatically tag each upload and then to be able to delete a specific upload.
I'm waiting for this feature to be released so if I do something bad while uploading I have some tools to revert my errors.
https://github.com/simulot/immich-go/pull/512
